lib/deaf_grandma.rb

```diff
@@ -19,9 +19,16 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    if(input == input.upcase)
+      if(input == "BYE BYE BYE")
```

Comment:
I think the requirements are inputing "BYE" to `DeafGrandma#speak` 3 times. It'll change the code here as you'll probably need to use the `@bye_counter`